# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmFrameIter.cpp
# Commit: 67ae30727ed8
# Full Hash: 67ae30727ed835cf73153a4ee8f0c9c7931aab09
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2021-12-23 09:34:36
# Regressor Bug: 1642412
# File Overlap Count: 7
# Description:
#   Bug 1642412 - wasm: Remove unnecessary code from Instance for inline table methods. r=lth
#   
#   This commit removes support for anyref in Instance::tableGet/tableSet,
#   making it clear that these functions are only used for function tables.
#   Instance::tableSize is removed as no longer needed.
# ==============================================================================

diff -r 90fa382b5a20 -r 67ae30727ed8 js/src/wasm/WasmFrameIter.cpp
--- a/js/src/wasm/WasmFrameIter.cpp	Wed Dec 22 17:36:30 2021 +0000
+++ b/js/src/wasm/WasmFrameIter.cpp	Wed Dec 22 17:36:30 2021 +0000
@@ -1539,16 +1539,14 @@
       return "call to native table.fill function";
     case SymbolicAddress::ElemDrop:
       return "call to native elem.drop function";
-    case SymbolicAddress::TableGet:
+    case SymbolicAddress::TableGetFunc:
       return "call to native table.get function";
     case SymbolicAddress::TableGrow:
       return "call to native table.grow function";
     case SymbolicAddress::TableInit:
       return "call to native table.init function";
-    case SymbolicAddress::TableSet:
+    case SymbolicAddress::TableSetFunc:
       return "call to native table.set function";
-    case SymbolicAddress::TableSize:
-      return "call to native table.size function";
     case SymbolicAddress::RefFunc:
       return "call to native ref.func function";
     case SymbolicAddress::PreBarrierFiltering: